pipeline/action.py

```diff
@@ -0,0 +1,40 @@
+'''
+Schema of session information.
+'''
+import datajoint as dj
+from pipeline import reference, subject
+
+schema = dj.schema('gao2018_action')
+
+@schema
+class Weighing(dj.Manual):
+    definition =  """
+    -> subject.Subject
+    ---
+    weight_before: float   # in grams
+    weight_after: float    # in grams
+    """
+
+@schema
+class WhiskerConfig(dj.Manual):
+    definition = """
+    -> subject.Subject
+    ---
+    -> reference.Whisker
+    """
+
+@schema
+class VirusInjection(dj.Manual):
+    definition = """
```

Comment:
I think I would like to see the primary key here as 
```
-> subject.Subject
injection : smallint unsigned   # injection number for the subject
```
and include the other references among the secondary attributes.